_layouts/account-page.html

```diff
@@ -0,0 +1,82 @@
+<!doctype html>
+
+{% include head.html %}
+
+<body>
+  {% include header-extended.html %}
+
+  <main id="main-content">
+	<section class="bg-primary usa-section border-bottom-05 border-secondary-dark">
+	  <div class="grid-container margin-y-5">
+	    <div class="tablet:grid-col-9">
+	      <h1 class="text-base-lightest">Hello, Carrington</h1>
+	      <p class="usa-intro text-base-lightest">Manage your orders and account details</p>
+	    </div>
+	  </div>
+	</section>
+
+	<div class="grid-container margin-y-5">
+	  <div class="grid-row grid-gap">
+	  	{% for info in page.cards %}
+	  	  <div class="tablet:grid-col-6 usa-card">
+            <div class="usa-card__container">
```

Comment:
Check out the card component again. I think there's a wrapper "group" class or something that avoid the extra space on the left side.